tests/password_manager/test_about_logins_edit_password.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+from modules.page_object import AboutLogins
+from modules.util import BrowserActions
+from faker import Faker
+
+
+@pytest.fixture()
+def test_case():
+    return "2241083"
+
+
+def test_about_logins_edit_password(driver_and_saved_logins, faker: Faker):
+    """
+    C2241083 - Verifies editing the password of an existing login entry in the about:logins page.
+    """
+    (driver, usernames, logins) = driver_and_saved_logins
+    ba = BrowserActions(driver)
+    about_logins = AboutLogins(driver).open()
+    ba.clear_and_fill(about_logins.get_element("login-filter-input"), usernames[-1])
+    login_results = about_logins.get_elements("login-list-item")
+    first_login = login_results[0]
+    driver.execute_script(
```

Comment:
We generally don't want arbitrary `execute_script` calls in the tests, it makes them difficult to maintain in the future. Please convert selectors to entries in `modules/data/about_logins.components.json`, add methods to the relevant Page Object in `modules/page_object_about_pages.py` if necessary, and execute commands from that object or the BasePage object in order to complete the test. Thanks!